### DISCUSSION TITLE
[Art competition] XSS

### DISCUSSION THREAD

**Discussion by TriangleYJ** [prosyslab-classroom/cs348-information-security#409] [Art competition]
![DALL·E 2023-05-27 17 09 33 - A mischievous thief unlocks an American house door, hindered by a chain latch  A radiant light emanates through the narrow gap, revealing a vibrant in](https://github.com/prosyslab-classroom/cs348-information-security/assets/9069813/3ff976da-1ce3-4a52-889a-9e227df1c4f2)

## Short Description
For me, XSS means "A mischievous thief unlocks an American house door, hindered by a chain latch. A radiant light emanates through the narrow gap, revealing a vibrant interior. Surrounding the door, a whimsical, magical forest blooms. Create a digital art-style image capturing the thief, the partially open door with its alluring light, and the enchanting surroundings of the vibrant forest.".

## Long Description (up to 3 sentences)
I've tried XSS attacks on various KAIST sites, and I've figured that XSS attacks were worked at some sites because they don't sanitize completely on the server for the malicious code. However, even if the XSS attack works, the XSS itself may not be a critical threat, for example, not able to steal cookies immediately because of the HttpOnly option for important cookies, rejected by the CORS policy, or new version of browsers blocks some potential threatening features. I imagined that a thief can open the door of other person's house through door lock hacking, and this alone could be a threat in some cases, but unless I find another way to release the inside chain latch, the thief can't steal and just watch inside the house through the narrow gap, which is the most similar to what I felt.

**Comment by KihongHeo** [comment]
Very nice. I wonder how much fine-tuning on the prompt you tried.
